content/15_PublishingOperators.md

```diff
@@ -0,0 +1,409 @@
+---
+title: Publishing operators
+---
+
+# Publishing operators
+
+Hot sources need to be able to deliver events to multiple subscribers. While we can implement the subscriber tracking ourselves, it can be easier to write a degenerate source that works only for a single subscriber, and then to use one of Rx's _multicast_ operators to publish it as a multi-subscriber hot source. The example in ["Representing Filesystem Events in Rx"](03_CreatingObservableSequences.md#representing-filesystem-events-in-rx) used this trick, but as you'll see in this chapter there are a few variations on the theme.
```

Comment:
Really? Are you unfamiliar with its use as a technical term? (I thought it was a bog standard technical term, but perhaps it's a bit computer sciency and I hadn't realised.) Or is that you are familiar with that usage, but don't think it's the right thing here?

Reply:
I don't think I've seen that phrase before... hence why I flagged it. If I google it... the 6th result is... this page!

Reply:
It's not really "degenerate source" as a phrase, so much as the word "degenerate" applied to, well, anything really.

In a technical context it tends to mean something that technically just about qualifies as an example of something more complex, but only barely. An empty set is a degenerate set. Adding together no values at all to produce a grand total of zero is a degenerate example of calculating a sum total. The number zero can, strictly speaking, be considered to be a 14-degree polynomial, but it's one where all 14 coefficients are zero, so it's a degenerate 14-degree polynomial. A completely ordinary number qualifies, technically, as a kind of tensor, but one where all the things that make tensors interesting have effectively been turned down to zero, so although it does meet all the defining qualifications, it's a degenerate tensor.

It comes up in maths all the time because it can often be useful to treat something very simple as an example (albeit a degenerate example) of something more complex.

Reply:
...but since the term evidently isn't as widely understood as I thought, I've replaced it. (And the amount of text it took to replace it is the reason I used the word in the first place.)